tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -232,6 +236,705 @@ def test_bobby_profiler_user_workflow_multi_batch_row_count_range_rule_and_colum
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": 0.0,
+        "max_value": 250.0,
+        "strict_min": False,
+        "strict_max": False,
+        "mostly": 8.75e-1,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "auto": True,
+        "profiler_config": None,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+    with pytest.raises(AssertionError) as e:
+        # noinspection PyUnusedLocal
+        result = validator.expect_column_values_to_be_between(
+            column=column_name,
+            mostly=1.0,
+            result_format="SUMMARY",
+            include_config=True,
+            auto=False,
+            profiler_config=None,
+        )
+    assert "min_value and max_value cannot both be None" in str(e.value)
+
+    result = validator.expect_column_values_to_be_between(
+        column=column_name,
+        min_value=-52.0,
+        max_value=3004.0,
+        mostly=1.0,
+        result_format="SUMMARY",
+        include_config=True,
+    )
+    assert result.success
+    assert result.expectation_config["kwargs"] == {
+        "column": "fare_amount",
+        "min_value": -52.0,
+        "max_value": 3004.0,
+        "mostly": 1.0,
+        "result_format": "SUMMARY",
+        "include_config": True,
+        "batch_id": "90bb41c1fbd7c71c05dbc8695320af71",
+    }
+
+
+@pytest.mark.skipif(
+    version.parse(np.version.version) < version.parse("1.21.0"),
+    reason="requires numpy version 1.21.0 or newer",
+)
+@freeze_time("09/26/2019 13:42:41")
+def test_bobby_expect_column_values_to_be_between_auto_yes_default_profiler_config_yes_custom_profiler_config_yes(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    context: DataContext = bobby_columnar_table_multi_batch_deterministic_data_context
+
+    expectation_suite_name: str = f"tmp.profiler_suite_{str(uuid.uuid4())[:8]}"
+    try:
+        # noinspection PyUnusedLocal
+        suite = context.get_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+    except ge_exceptions.DataContextError:
+        suite = context.create_expectation_suite(
+            expectation_suite_name=expectation_suite_name
+        )
+        print(f'Created ExpectationSuite "{suite.expectation_suite_name}".')
+
+    batch_request: dict
+
+    validator: Validator
+
+    result: ExpectationValidationResult
+
+    custom_profiler_config: dict
+
+    # Use all batches, loaded by Validator, for estimating Expectation argument values.
+    batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    validator = context.get_validator(
+        batch_request=BatchRequest(**batch_request),
+        expectation_suite_name=expectation_suite_name,
+    )
+    assert len(validator.batches) == 3
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner Again -- this is a custom override, provided by users (who may find plain dictionary more convenient -- at least we have held this pattern for user-facing `batch_request` API).

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!